File: lab/lab05/echon.pl
#!/usr/bin/perl -w
die "Usage: ./echon.pl <number of lines> <string>\n" if @ARGV != 2;
die "./echon.pl: argument 1 must be a non-negative integer\n" if $ARGV[0] !~ /^\d+$/;
for($i=1;$i<=$ARGV[0];$i++){
  printf "$ARGV[1]\n";
}

File: 参考/comp9041-Q/cs9041/lab06/echon.py
#!/usr/bin/python3
from sys import *


if len(argv)!=3:
	print("Usage: {} <number of lines> <string>".format(argv[0]))
	exit()
if not argv[1].isdigit() or int(argv[1])!=float(argv[1]):
	print("{}: argument 1 must be a non-negative integer".format(argv[0]))
	exit()
for i in range(int(argv[1])):
	print(argv[2])